src/kili/services/label_import/__init__.py

```diff
@@ -82,5 +83,9 @@ def import_labels_from_dict(
     logger_params = LoggerParams(disable_tqdm=False, level=cast(LogLevel, "WARNING"))
```

Comment:
`disable_tqdm` was already defined here in the loggerParams and always set to False. You just need to change the False by the `disable_tqdm` given in the arguments. It is the one of the LoggerParams that will be used